src/databricks/labs/remorph/transpiler/execute.py

```diff
@@ -211,7 +211,7 @@ def morph(workspace_client: WorkspaceClient, config: MorphConfig):
     return status
 
 
-def _verify_workspace_client(workspace_client: WorkspaceClient) -> WorkspaceClient:
+def verify_workspace_client(workspace_client: WorkspaceClient) -> WorkspaceClient:
```

Comment:
add a doc string saying it is private and also add a TODO for us to come to back to this.

Reply:
I added to do in reconcile/execute.py. But anyway I have added [private] label and todo for transpile/execute.py file